lib/improver/metadata/utilities.py

```diff
@@ -31,10 +31,57 @@
 """General IMPROVER metadata utilities"""
 
 import hashlib
+import iris
 # Usage of pickle in this module is only for creation of pickles and hashing
 # the contents. There is no loading pickles which would create security risks.
 import pickle  # nosec
 
+import dask.array as da
+import numpy as np
+
+
+def create_new_diagnostic_cube(
+        name, units, coordinate_template, attributes=None, data=None,
+        datatype=np.float32):
```

Comment:
`dtype` is a widely used and understood name within numpy and related packages, we shouldn't change it.
```suggestion
        dtype=np.float32):
```

Reply:
I was worried that implied something about the whole cube, rather than just the data.  But will change.